### ISSUE TITLE
about the -O1 output

### ISSUE THREAD

**Issue by ljl000** [ogotoh/spaln#38]
Hi, I used spaln to align protein sequence to genomic dna sequence and I got the result like this 
<chr03:18371009-18394190 [1:23181]  ( 23181 - 1 ) - >1.faa [1:70]  ( 1 - 70 )
;C complement(join(22030..22242))
1.faa was my query and chr03:18371009-18394190 was the  target sequence coordinate from gff3.  
How can I explain the result above?
Did 23181 - 1 mean that my query align with the reverse target sequence?
what did C complement(join(22030..22242)) mean?

**Comment by ogotoh** [comment]
Yes, your query appears to match the reverse complementary strand of the genomic sequence.

;C complement(join(22030..22242)) indicates the exonic range of the genomic sequence that matches the query. In this example, no intron exists, so that only one range (pair of coordinates that flank the exon) is shown.  The format after ;C is the same as that of feature table of Genbank entries.

Osamu,
